src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -383,12 +385,16 @@ function FilterSidebarSearchTab({
                     {boundaryButton}
                 </div>
                 <div className="form__action">
-                    <a
-                        className="control-link"
-                        href="mailto:info@openapparel.org?subject=Reporting an issue"
-                    >
-                        Report an issue
-                    </a>
+                    {embed ? (
+                        <div />
```

Comment:
This is to maintain styling - having no component here caused the search/reset buttons to shift left. I could rewrite the styling code explicitly instead, if desired

Reply:
2 cents: As a rule of thumb, best to avoid spacer divs and make the styling resilient to (expected) DOM variations. But not a huge deal here, and it should disappear anyway when we do a more holistic update to the search form to accommodate additional functionality.